Dockerfile.ubi8

```diff
@@ -0,0 +1,21 @@
+# Build the manager binary
+FROM registry.access.redhat.com/ubi8/go-toolset:1.19 as builder
+
+# Copy in the go src
+WORKDIR /go/src/github.com/kserve/kserve
+COPY go.mod  go.mod
+COPY go.sum  go.sum
+
+RUN go mod download
+
+COPY cmd/    cmd/
+COPY pkg/    pkg/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOFLAGS=-mod=mod go build -a -o manager ./cmd/manager
+
+# Use distroless as minimal base image to package the manager binary
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6 
```

Comment:
we could even evaluate to use of ubi9.

if that is not possible, use the latest 8.x, IIRC, 8.9.

Reply:
devOps team is discussing this transition. before they decide, we should use ubi8